src/Login/Login.tsx

```diff
@@ -1,65 +1,75 @@
-import React, { useState } from 'react';
-import { Link } from 'react-router-dom';
-import './Login.scss';
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import "./Login.scss";
 
 export interface LoginProps {
-	username: string;
-	setUsername: any;
-	loggedIn: boolean;
-	setLoggedIn: any;
-}
+  username: string;
+  setUsername: Function;
```

Comment:
Added in TS type as Function for setUsername & setLoggedIn

Reply:
nice job fixing these!

Reply:
ahhhh, i see. thanks for figuring that out! i was scratching my head about what to type setUsername as for some dumb reason hah